ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -41,21 +44,44 @@ object Assertions {
     }
   }
 
+  /** A non-regex alternative to [[assertGrpcErrorRegex]] which just does a substring check.
+    */
+  def assertGrpcError(
+      t: Throwable,
+      expectedCode: Status.Code,
+      exceptionMessageSubstring: Option[String],
+      checkDefiniteAnswerMetadata: Boolean = false,
+  ): Unit =
+    assertGrpcErrorRegex(
+      t,
+      expectedCode,
+      exceptionMessageSubstring
+        .map(msgSubstring => Pattern.compile(Pattern.quote(msgSubstring))),
+      checkDefiniteAnswerMetadata,
+    )
+
   /** Match the given exception against a status code and a regex for the expected message.
-    *      Succeeds if the exception is a GrpcException with the expected code and
-    *      the regex matches some part of the message or there is no message and the pattern is
-    *      None.
+    * Succeeds if the exception is a GrpcException with the expected code and
+    * the regex matches some part of the message or there is no message and the pattern is
+    * None.
     */
   @tailrec
-  def assertGrpcError(t: Throwable, expectedCode: Status.Code, optPattern: Option[Pattern]): Unit =
+  def assertGrpcErrorRegex(
+      t: Throwable,
+      expectedCode: Status.Code,
+      optPattern: Option[Pattern],
+      checkDefiniteAnswerMetadata: Boolean = false,
```

Comment:
I think this naming is a bit obtuse. I was expecting to check something about the definitive answer. 
Maybe renaming to something like `assertDefinitiveAnswerIsReceived`

Reply:
Cosmetics, I can change if you insist :) This one is shorter :)